### PR TITLE
feat(server): add latency statistics for lua script calls

### LINKED ISSUE #754: allow listing lua scripts
Often, I need to see what scripts are loaded into DF. Redis 7 introduced API  (FUNCTION) that does it:
https://redis.io/commands/function-list/

We should implement part of it (without LIBRARYNAME option).

In addition, we should allow seeing latency stats for each of the scripts.
Update: decided against "FUNCTION LIST" because "FUNCTION" in redis does not cover "eval/script" API.

### PR DESCRIPTION
**Pull request by romange**
Also provide new subcommands "script list" and "script latency" Fixes #754

Signed-off-by: Roman Gershman <roman@dragonflydb.io>

### CODE REVIEW

**Review by dranikpg — DISMISSED** [review]
You missed a few moves here and there when passing the scripts around 👀 , but I assume its not important at all

**Comment by romange** [comment]
can you detail what I missed? :)

File: src/server/dflycmd.cc [hunk 1]
```diff
@@ -455,7 +455,11 @@ void DflyCmd::FullSyncFb(FlowInfo* flow, Context* cntx) {
 
   if (saver->Mode() == SaveMode::SUMMARY) {
     auto scripts = sf_->script_mgr()->GetLuaScripts();
-    ec = saver->SaveHeader(scripts);
+    StringVec script_bodies;
+    for (const auto& script : scripts) {
+      script_bodies.push_back(script.second);
+    }
+    ec = saver->SaveHeader(script_bodies);
```
Comment (dranikpg) [inline src/server/dflycmd.cc:462]: You get the scripts by value here and proceed to build a new container by copying the scripts itself

File: src/server/server_family.cc [hunk 2]
```diff
@@ -876,6 +876,15 @@ GenericError ServerFamily::DoSave(bool new_version, Transaction* trans) {
     }
   };
 
+  auto get_scripts = [this] {
+    auto scripts = script_mgr_->GetLuaScripts();
+    StringVec script_bodies;
+    for (const auto& script : scripts) {
+      script_bodies.push_back(script.second);
+    }
+    return script_bodies;
```
Comment (dranikpg) [inline src/server/server_family.cc:885]: And here

File: src/server/dflycmd.cc (hunk 1 above)
Comment (romange) [reply src/server/dflycmd.cc]: fixed

File: src/server/server_family.cc (hunk 2 above)
Comment (romange) [reply src/server/server_family.cc]: fixed

**Review by dranikpg — APPROVED** [review]

**Pull request merged**